packages/atomic/cypress/integration/facets/facet/facet.cypress.ts

```diff
@@ -987,9 +963,9 @@ describe('Facet v1 Test Suites', () => {
       const index = [0, 1, 2];
       function setupSelectedMulitpleFacets() {
         setupBreadboxWithFacet();
-        index.forEach((i: number) => {
-          selectIdleCheckboxValueAt(FacetSelectors, i);
-          cy.wait(TestFixture.interceptAliases.Search);
+        index.forEach((position, i) => {
+          selectIdleCheckboxValueAt(FacetSelectors, position);
+          BreadboxSelectors.breadcrumbButton().should('have.length', i + 1);
```

Comment:
At some point, I had flakiness here due to a bad test and I added this assertion. I didn't remove it after fixing the test, since I think it's still relevant. Although we are selecting facet values, what we're concerned about in this section of the tests is which breadcrumbs are rendered, and I want to make sure breadcrumbs were rendered too.